### DISCUSSION TITLE
Add another repo content inside this theme

### DISCUSSION THREAD

**Discussion by cobrasanjay1** [cotes2020/jekyll-theme-chirpy#2032] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to add a new category called portfolio in sidebar and, in that category it should display index.html of this repo https://github.com/MZaFaRM/portfolio

### Operations you have already tried

i created a new category called portfolio in in _tabs dir
i created portfolio.html in _layouts 
I don't know how to link my index.html in that sidebar with chirpy theme

**Comment by kungfux** [comment]
If you want to keep portfolio separately but still incorporate it inside the chirpy, try the following:

1. Create new tab e.g. `_tabs/portfolio.md`.
2. Use [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) to load content from other source inside portfolio tab.

```md
---
icon: fas fa-info-circle
order: 5
---

<iframe src="https://raw.githubusercontent.com/MZaFaRM/portfolio/refs/heads/main/index.html"></iframe>
```

However, if portfolio is not available by its own URL, you will experience issues loading resources like JS/CSS and so it will require modifications for such specific usage.
